apps/ledger-live-mobile/e2e/page/trade/swap.page.ts

```diff
@@ -22,49 +23,62 @@ export default class SwapFormPage {
   termsAcceptButton = () => getElementById("terms-accept-button");
   termsCloseButton = () => getElementById("terms-close-button");
 
-  openViaDeeplink() {
-    return openDeeplink(baseLink);
+  async openViaDeeplink() {
+    await openDeeplink(baseLink);
   }
 
-  navigateToSwapForm() {
-    return tapByElement(this.swapFormTab());
+  async expectSwapPage() {
+    await expect(this.swapFormTab()).toBeVisible();
   }
 
-  navigateToSwapHistory() {
-    return tapByElement(this.swapHistoryTab());
+  async navigateToSwapForm() {
```

Comment:
this method is not in use anywhere, do we need to keep it anyway?

Reply:
It's part of a path that should be tested.